storage/statedb/database.go

```diff
@@ -336,14 +336,9 @@ func getTrieNodeCacheSizeMiB() int {
 		return 0
 	} else if totalPhysicalMemMiB < 20*1024 {
 		return 1 * 1024 // allocate 1G for small memory
+	} else {
+		return 10 * 1024 // allocate 10G for large memory (>20G)
 	}
```

Comment:
I think 10GB is too big for 20GB ~ 30GB memory. 
How about the following?
```suggestion
	} else if totalPhysicalMemMiB < 30*1024 {
		return 6 * 1024 // allocate 6G for medium memory
	} else {
		return 10 * 1024 // allocate 10G for large memory (>20G)
	} 
```

Reply:
Ah, I've made a significant mistake. Thank you for pointing that out. The only minor adjustment I'd like to make is to reduce the number from 6 to 3. Does that sound acceptable to you? (becuause it seems to jump sharply)

Reply:
I tried to maintain the original value is possible. Originally, 6GB is assigned for 20GB of physical memory (20GB * 0.3).

Reply:
I see. thanks.